src/components/SingleVault/index.js

```diff
@@ -54,29 +54,30 @@ function SingleVaultPage({value}){
 	}, [vault, provider]);
 
 	const getStrategyApr = useCallback((strategy) => {
-		if(!strategy.succeded){
+		if(!strategy.succeded || strategy.beforeDebt === 0){
```

Comment:
if debt == 0, then apr should be infinity no? Can we provide this info? I sometimes simulate harvests and this is good because it means that we left some profit behind and we need to harvest to finally close the strat.

Reply:
Makes sense. OK I changed the logic to show "∞" when strategy debt is zero and estimated strategy assets is still greater than zero. Does that work? https://github.com/yearn/dashboard_ui/pull/86/commits/854ba23fd06ec39b641d6c21e79768b1fbb45607

For example:
![image](https://user-images.githubusercontent.com/89237203/185722767-d96fc50a-e702-40c7-b754-0fe6de1d4dda.png)

(note, I mocked the data for that. The 88 snx strategy does actually have debt)

Reply:
Great! Let's send it to prod